src/heroes/dva/baby.opy

```diff
@@ -3,16 +3,8 @@
 
 rule "Set baby D.va Ult charge to 0":
     @Event eachPlayer
-    @Hero dva
-    @Condition eventPlayer.isInAlternateForm()
-
-    eventPlayer.setUltCharge(0)
-
-
-rule "[Echo] Set baby D.va Ult charge to 0":
-    @Event eachPlayer
-    @Hero echo
-    @Condition eventPlayer.getHeroOfDuplication() == Hero.DVA
+    @Condition eventPlayer.hero == Hero.DVA
     @Condition eventPlayer.isInAlternateForm()
 
+    #don't you already start with 0 ult charge as baby?
```

Comment:
There was a problem with one of the subroutines that checks what your previous Ult charge was for the Tank reduced Ult charge passive. Baby d.va would have the ult charge she had as big d.va